dp/pipeline/epc_arp.c

```diff
@@ -301,7 +301,7 @@ struct ether_addr broadcast_ether_addr = {
 	.addr_bytes[4] = 0xFF,
 	.addr_bytes[5] = 0xFF,
 };
-static const struct ether_addr null_ether_addr = {
+static const struct ether_addr null_ether_addr __rte_unused = {
```

Comment:
@suresh-marikkannu Should we keep the `null_ether_addr` with `__rte_unused` or remove altogether?

Reply:
@krsna1729 , It is not used anywhere, so we can remove it completely. Also struct ether_addr broadcast_ether_addr (just above the static const struct ether_addr null_ether_addr) also not used. I think we can remove both.
@somnathchakrabarti , Any suggestions?

Reply:
@suresh-marikkannu i think it tagged only this one since this declared static and was not being used in this file?

Reply:
@suresh-marikkannu @somnathchakrabarti removing both builds fine.